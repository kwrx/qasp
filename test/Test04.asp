%Run with options: A=2 a=1 c=1 E=2 e=1 w=2 out=PrintAspQ 
%@forall
x_1 | nx_1.
x_2 | nx_2.
%@exists
x_3 | nx_3.
x_4 | nx_4.
x_5 | nx_5.
x_6 | nx_6.
x_7 | nx_7.
%@constraint
:- not x_3, not x_4, x_5.
:- x_3, not x_5.
:- x_4, not x_5.
:- not x_5.
