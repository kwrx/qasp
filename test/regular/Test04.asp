% https://en.wikipedia.org/wiki/Boolean_satisfiability_problem
% ∀x ∀y ∃z (x ∨ y ∨ z) ∧ (¬x ∨ ¬y ∨ ¬z)

%@forall
x | nx.
%@forall
y | ny.
%@exists
z | nz.
%@constraint
:- nx, ny, nz.
:- x, y, z.


%% Result: COHERENT