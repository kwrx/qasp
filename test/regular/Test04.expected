QASP 1.0

{x}
{nx}
